### DISCUSSION TITLE
allow processing the (json-) body of http error responses

### DISCUSSION THREAD

**Discussion by zyro23** [grafana/grafana-infinity-datasource#1086] [Ideas]
currently, it seems that http error responses (i.e. status >= 400) are propagated as errors with an accompanying error message containing the status code, however the body is not relayed/processed.

it would be nice to have an option to process the body of error responses as well (and relay the http status code, ref. #93).

thanks!

**Comment by yesoreyeram** [comment]
Will be fixed in https://github.com/grafana/grafana-infinity-datasource/pull/1076

**Reply by zyro23** [reply]
awesome. much appreciated!
